### PR TITLE
fix: remove anti-pattern from RepoSubmission

### LINKED ISSUE #317: Remove tailwinds anti-pattern in RepoSubmission.tsx
Context is on the tracking issue - https://github.com/open-sauced/hot/issues/267

This issue only needs changes to this file.

### PR DESCRIPTION
**Pull request by NsdHSO**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Removed anti-pattern with px. 

## Related Tickets & Documents

closes #317 

## Mobile & Desktop Screenshots/Recordings
old 
image

new 
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by mtfoley** [comment]
Two nits:
- for PR title I'd suggest "fix: ..." instead of "style: ..."
- for PR description I'd suggest a before and after screenshot of the affected account part of the app

**Comment by NsdHSO** [comment]
> Two nits:
> 
> * for PR title I'd suggest "fix: ..." instead of "style: ..."
> * for PR description I'd suggest a before and after screenshot of the affected account part of the app

Thanks for advice, it's a good thing.

File: src/components/RepoSubmission.tsx [hunk 1]
```diff
@@ -67,25 +67,25 @@ const RepoSubmission = ({ isFormOpen, handleFormOpen }: RepoSubmissionProps): JS
   return (
     <div
       ref={submissionRef}
-      className="fixed top-[60px] right-[120px] flex items-end flex-col gap-[10px] submission-form z-10"
+      className="fixed top-[3.75rem] right-[7.5rem] flex items-end flex-col gap-2.5 submission-form z-10"
```
Comment (bdougie) [inline src/components/RepoSubmission.tsx:70]: why the rems?

Comment (NsdHSO) [reply src/components/RepoSubmission.tsx]: Tailwind does not have 60px, and I said to translate them in rem. but still custom. But I put them in standard classes from a tailwind.

Comment (NsdHSO) [reply src/components/RepoSubmission.tsx]: with standard classes 
image

Comment (0-vortex) [reply src/components/RepoSubmission.tsx]: remove "left" entirely and use "right" instead - it's supposed to be 5 pixels from the right, fixing that bug enables tailwind sizing

Comment (NsdHSO) [reply src/components/RepoSubmission.tsx]: It is ok like that 
image

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Review by bdougie — APPROVED** [review]

**Pull request merged**